doc/subversion_and_git_integration.md

```diff
@@ -33,66 +33,86 @@ On that page, enable the "Enable WS for repository management" setting and gener
 for get to save the settings). We need that API key later in our apache config.
 
 Find a place to store the repositories. For this guide we assume that you put your svn repositories in
-/srv/openproject/svn . All things in that repository should be accessible by the apache system user and
-by the user running your openproject server. 
+/srv/openproject/svn and your git repositories in /srv/openproject/git .
+All things in that repository should be accessible by the apache system user and by the user running your openproject server. 
```

Comment:
I've referenced the configuration [in the reposman PR]( https://github.com/oliverguenther/openproject/blob/feature/deprecate_reposman/doc/subversion_and_git_integration.md), since this PR doesn't really depend on any configuration made there. This only concerns managed repositories, but you could still serve existing repositories in Apache and only link them through OpenProject